polaris-core/src/main/java/org/apache/polaris/core/config/FeatureConfiguration.java

```diff
@@ -201,4 +201,11 @@ protected FeatureConfiguration(
                   + " requires experimentation in the specific deployment environment")
           .defaultValue(100 * EntityWeigher.WEIGHT_PER_MB)
           .buildFeatureConfiguration();
+
+  public static final FeatureConfiguration<Boolean> ENABLE_STREAMING_TABLE_METADATA =
```

Comment:
Instead of just an enable/disable, can we have something like `STREAMING_TABLE_METADATA_MIN_SIZE`?